patches/chrome-install_static-install_modes.cc.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/chrome/install_static/install_modes.cc b/chrome/install_static/install_modes.cc
+index 42d9966254758305e0dc999b186aa7abe1ff762a..147dd51211c889bdcbde29ae0537a6e17d045dd8 100644
+--- a/chrome/install_static/install_modes.cc
++++ b/chrome/install_static/install_modes.cc
+@@ -12,6 +12,21 @@ std::wstring GetUnregisteredKeyPathForProduct(const wchar_t* product) {
+   return std::wstring(L"Software\\").append(product);
+ }
+ 
++#if defined(BRAVE_CHROMIUM_BUILD)
```

Comment:
override with undefine/redefine

Reply:
can't do for functions that are used by other functions in the same file.

Reply:
you can do it for functions that are used by other functions in the same file, but you would also have to do the functions that use that function. You can just only do the functions that use that function I think in this case.

Reply:
Left some comment for the record.
This is the reason why patching technique is used instead of overriding.
In case of `install_modes.cc`, override technique causes modifying all functions in that file and can't reuse upstream functions. It is similar to copying technique.
IMO, patching technique is more better to copying for future rebase.
I thought def/undef technique is *only* useful when we can re-use upstream implementation